src/_data/events.json

```diff
@@ -0,0 +1,13 @@
+{
+  "upcoming": [
```

Comment:
Would it make sense to just pull this at build time from Discord?

Reply:
We certainly could. It would likely mean we need a Discord app + API key created for it. Did we already have that somewhere? And if it is can you add it as secret env vars in the GitHub Action secrets settings for this repo?